scripts/build_data.py

```
@@ -7906,8 +7912,15 @@ def sanitize_place_page_display_name(value: Any) -> str | None:
     "went",
 )
 PLACE_PAGE_REVIEW_DESCRIPTION_MARKERS = (
+    "boy was it worth",
+    "definitely recommend this place",
+    "great experience overall",
     "highly recommended",
+    "i've tasted",
+    "i’ve tasted",
```

Ruff (0.15.12)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)